docs/writing-tests/interaction-testing.md

```diff
@@ -75,6 +121,33 @@ Since Storybook is a webapp, anyone with the URL can reproduce the error with th
 
 Streamline interaction testing further by automatically [publishing Storybook](../sharing/publish-storybook.md) in pull requests. That gives teams a universal reference point to test and debug stories.
 
+## Execute tests with the test-runner
+
+Storybook only runs the interaction test when you're viewing a story. Therefore, you'd have to go through each story to run all your checks. As your Storybook grows, it becomes unrealistic to review each change manually. Storybook [test-runner](https://github.com/storybookjs/test-runner) automates the process by running all tests for you. To execute the test-runner, open a new terminal window and run the following command:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-test-runner-execute.yarn.js.mdx',
+    'common/storybook-test-runner-execute.npm.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+![Interaction test with test runner](./storybook-interaction-test-runner-loginform-optimized.png)
+
+<div class="aside">
+
+💡 If you need, you can provide additional flags to the test-runner. Read the [documentation](./test-runner.md#cli-options) to learn more.
+
+</div>
+
+## Automate
+
+Once you're ready to push your code into a pull request, you'll want to automatically run all your checks using a Continuous Integration (CI) service before merging it. Read our [documentation](./test-runner.md#set-up-ci-to-run-tests) for a detailed guide on setting up a CI environment to run tests.
```

Comment:
```suggestion
Once you're ready to push your code into a pull request, you'll want to automatically run all your checks using a Continuous Integration (CI) service before merging it. Read our detailed guide on [setting up a CI environment to run tests](./test-runner.md#set-up-ci-to-run-tests).
```

Reply:
@kylegach I phrased like so to also avoid a situation where the documentation breaks and generates an extremely long space. For instance, if I apply your suggestion here's what happens when the docs render locally:
![broken-paragraph-docs](https://user-images.githubusercontent.com/22988955/163883970-31d69e19-a4ba-49fb-b434-4e2f5ba1d1dc.png)

If you don't mind, I'll leave it as is for the time being.

Reply:
Yeah, that's fine. It's not a big deal. Seems odd that our links don't wrap, but that's a separate issue.